ci/pipelines/Release.jenkinsfile

```diff
@@ -55,35 +66,25 @@ pipeline
 			{
 				script
 				{
-					def configurations = ['NewtonDebug', 'NewtonRelease', 'SnowDebug', 'SnowRelease']
-					for(int i=0; i < configurations.size(); i++)
-					{
-						stage('Build ' + configurations[i])
-						{
-							def props = readProperties file:"ci/pipelines/${configurations[i]}.properties"							
-							props.each { propKey, propValue -> println "${propKey}: ${propValue}" }
-
-							sh "mkdir build-${configurations[i]}"
-							dir("build-${configurations[i]}")
+					archs.each
+					{ arch ->
+						configs.each
+						{ config ->
+							stage('Build ${config}|${arch}')
 							{
-								sh "conan install .. -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} -o openssl=${props.conanOpenSSLVersion}"
-								sh "cmake .. -G '${props.cmakeVS}' -A ${props.cmakePlatform}"
-								sh "cmake --build . --config ${props.cmakeConfiguration}"
-								sh "conan export-pkg ../conanfile.py JWTUtils/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} -o openssl=${props.conanOpenSSLVersion} --force"
-								sh "conan test ../test_package/conanfile.py JWTUtils/${version}@systelab/${channel} -s build_type=${props.conanBuildType} -s compiler.toolset=${props.conanCompilerToolset} -s arch=${props.conanArch} -o gtest=${props.conanGTestVersion} -o openssl=${props.conanOpenSSLVersion}"
-								sh "bin/${props.conanBuildType}/JWTUtilsTest.exe --gtest_output=xml:JWTUtilsTest.xml"
-							}
+								def buildFolder = "build-${config}-${arch}"
+								bat "conan install . -if ${buildFolder} --profile=${profile} -s arch=${arch} -s build_type=${config}"
```

Comment:
I really think specific versions for dependencies should be overriden here. Then the conanfile would not vary in case of an upgrade as it could be defined with a version range. 
This approach doesn't work well with conan test. Or at least I haven't managed. Dependencies cannot be specified to be overridden in this step and fails if no package of the latest possible version is missing.

Reply:
I don't see a problem in having set in the conan file a specific version (and/or revision) because you don't know whether a range will work or not (though it should if they preserve ABI compatibility), you can always override from the outside.

Reply:
More of a hazard when the package is in conancenter, but yes.